router/fastdp.go

```diff
@@ -209,25 +209,25 @@ func (lock *fastDatapathLock) relock() {
 	}
 }
 
-// Bridge bits
+// InjectorConsumer bits
 
-type fastDatapathBridge struct {
+type fastDatapathInjectorConsumer struct {
```

Comment:
Not sure about this name, as besides injecting / consuming, it does bridging (see `bridge` method of the struct) as well.